tests/test-dirs/occurrences/issue1404.t

```diff
@@ -88,6 +88,6 @@ locate position 2:1 returns the definition of [(+)]
     "file": "lib/ocaml/stdlib.mli",
     "pos": {
       "line": 344,
-      "col": 0
+      "col": 9
```

Comment:
What's the context for this?

Reply:
There's actually no net change here. On main this is `9` (`9` is the correct value), but due to the cms thing, the column here changed to `0`. When I changed the prioritization order, this changed back to 9. I just happened to promote and commit this test before making the prioritization change (see f1a8cdbd62abd4aaea8daf7fbce85a37189aa9d7).